core/src/main/scala/higherkindness/mu/rpc/srcgen/avro/AvroSrcGenerator.scala

```diff
@@ -103,6 +103,7 @@ final case class AvroSrcGenerator(
       })
       .map(generateFrom(_, serializationType))
 
+  //TODO: implement this using higherkindness.skeuomorph.mu.Protocol.fromAvroProtocol, as is done in ProtoSrcGenerator
```

Comment:
This is probably part of https://github.com/higherkindness/sbt-mu-srcgen/pull/101, right?

Reply:
Oh, I was just making a note of the discrepancy and forgot there was an issue to address this. :-)

Reply:
I'm rebasing #101 off of this in anticipation of yours being merged first :D